types/is-docker/index.d.ts

```diff
@@ -0,0 +1,6 @@
+// Type definitions for is-docker 1.1
+// Project: https://github.com/sindresorhus/is-docker#readme
+// Definitions by: Yash Kulshrestha <https://github.com/YashdalfTheGray>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export default function isDocker(): boolean;
```

Comment:
It looks like `require("is-docker")` is itself a function, not an object with a `default` property. So this should be:
```ts
declare function isDocker(): boolean;
export = isDocker;
```
If you want to keep the current import style you could add `"esModuleInterop": true,` to `tsconfig.json`, or you could change the import to `import isDocker = require("is-docker");`.

Reply:
Alright, I'll make the change! Thanks! Still new to trying and matching types to code.

Reply:
Done.

Reply:
The export should still be changed -- import doesn't have to be changed because `--esModuleInterop` allows you to default-import an `export =`, but for users without that flag we should use `export =` for accuracy.